torchmetrics/functional/classification/cohen_kappa.py

```diff
@@ -0,0 +1,99 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Optional
+
+import torch
+
+from torchmetrics.functional.classification.confusion_matrix import _confusion_matrix_compute, _confusion_matrix_update
+
+
+def _cohen_kappa_update(
+        preds: torch.Tensor,
+        target: torch.Tensor,
+        num_classes: int,
+        threshold: float = 0.5
+) -> torch.Tensor:
+    return _confusion_matrix_update(preds, target, num_classes, threshold)
```

Comment:
```suggestion
_cohen_kappa_update = _confusion_matrix_update
```

no need for all the other lines :)